resources/doctrines/coldwar.yaml

```diff
@@ -0,0 +1,32 @@
+name: coldwar
+cap: true
+cas: true
+sead: true
+strike: true
+antiship: true
+rendezvous_altitude: 22000
```

Comment:
Probably best to add the units to the key name for these. It's obvious when it's in Python and it's using `Distance` instead of int/float, but here the author would need to consult the docs to figure out the units. `rendezvous_altitude_ft`, for example. (maybe even `rendezvous_altitude_ft_msl`?)

Alternatively (but more work, mostly in the form of handling malformed inputs), you could do something like `rendezvous_altitude: 22000 ft MSL`.

Reply:
Units have been added. I was thinking along the same lines but other YAML keys didn't seem to have units in them so I went with consistency. Agree it's a good idea to introduce this precedent.

Reply:
Ha, I almost said "I bet I haven't followed own advice here, but..."